specification/azsadmin/resource-manager/user-subscriptions/Microsoft.Subscriptions/preview/2015-11-01/Subscriptions.json

```diff
@@ -0,0 +1,412 @@
+                "type": "string",
+                "enum": [
+                    "NotDefined",
+                    "Enabled",
+                    "Warned",
+                    "PastDue",
+                    "Disabled",
+                    "Deleted"
+                ],
+                "x-ms-enum": {
+                    "name": "SubscriptionState",
+                    "modelAsString": true
+                }
+            },
+            "Operation": {
+                "description": "Supported REST operation.",
+                "type": "object",
+                "properties": {
+                    "display": {
+                        "description": "Display information for the operation.",
+                        "type": "object",
+                        "properties": {
+                            "description": {
+                                "description": "Description for the operation.",
+                                "type": "string",
+                                "readOnly": true
+                            },
+                            "operation": {
+                                "description": "Display name of the operation.",
+                                "type": "string",
+                                "readOnly": true
+                            },
+                            "provider": {
+                                "description": "Resource provider display name.",
+                                "type": "string",
+                                "readOnly": true
+                            },
+                            "resource": {
+                                "description": "Resource type display name related to the operation.",
+                                "type": "string",
+                                "readOnly": true
+                            }
+                        }
+                    },
+                    "name": {
+                        "description": "Name of the operation being performed on an object.",
+                        "type": "string",
+                        "readOnly": true
+                    },
+                    "origin": {
+                        "description": "Origin for the operation.",
+                        "type": "string",
+                        "readOnly": true
+                    }
+                }
+            },
+            "OperationList": {
+                "description": "List of supported operations.",
+                "type": "object",
+                "properties": {
+                    "value": {
+                        "description": "List of supported operations.",
+                        "type": "array",
+                        "items": {
+                            "$ref": "#/definitions/Operation"
+                        }
+                    }
+                }
+            }            
+        },
+        "parameters": {
+            "SubscriptionIdParameter": {
```

Comment:
Not being referenced.  If you don't want it to be global change it to being passed in by a method.

Reply:
removed